LSP-ruff.sublime-settings

```diff
@@ -28,8 +25,6 @@
             "lint.preview": null,
             // Sets the tracing level for the extension.
             "logLevel": "error",
```

Comment:
Most likely some other options like this one also have no effect in ST.
But someone can look at it later.